workers/tests/integrationtests/test_api_regions.py

```diff
@@ -21,6 +21,41 @@ def test_get_available_regions(self):
         self.assertEqual(response.status_code, 200)
         self.assertTrue(geojson.loads(response.content).is_valid)
 
+    def test_get_list_indicators(self):
+        url = "/indicatorNames"
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsInstance(geojson.loads(response.content), dict)
+        self.assertIsNotNone(geojson.loads(response.content))
+
+    def test_get_list_layers(self):
+        url = "/layerNames"
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsInstance(geojson.loads(response.content), dict)
+        self.assertIsNotNone(geojson.loads(response.content))
+
+    def test_get_list_datasets(self):
+        url = "/datasetNames"
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsInstance(geojson.loads(response.content), dict)
+        self.assertIsNotNone(geojson.loads(response.content))
+
+    def test_get_list_fid_field(self):
+        url = "/FidFields"
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsInstance(geojson.loads(response.content), dict)
+        self.assertIsNotNone(geojson.loads(response.content))
+
+    def test_get_list_reports(self):
+        url = "/reportNames"
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsInstance(geojson.loads(response.content), dict)
+        self.assertIsNotNone(geojson.loads(response.content))
```

Comment:
Testing for None is not necessary anymore after testing if the object is of instance dict:

```suggestion
    def test_get_list_indicators(self):
        url = "/indicatorNames"
        response = self.client.get(url)
        self.assertEqual(response.status_code, 200)
        self.assertIsInstance(geojson.loads(response.content), dict)

    def test_get_list_layers(self):
        url = "/layerNames"
        response = self.client.get(url)
        self.assertEqual(response.status_code, 200)
        self.assertIsInstance(geojson.loads(response.content), dict)

    def test_get_list_datasets(self):
        url = "/datasetNames"
        response = self.client.get(url)
        self.assertEqual(response.status_code, 200)
        self.assertIsInstance(geojson.loads(response.content), dict)

    def test_get_list_fid_field(self):
        url = "/FidFields"
        response = self.client.get(url)
        self.assertEqual(response.status_code, 200)
        self.assertIsInstance(geojson.loads(response.content), dict)

    def test_get_list_reports(self):
        url = "/reportNames"
        response = self.client.get(url)
        self.assertEqual(response.status_code, 200)
        self.assertIsInstance(geojson.loads(response.content), dict)
```